app.py

```diff
@@ -1625,45 +1650,33 @@ def sendEmail(project_id, evaluation_name, show_score):
     current_record.num_of_finished_tasks = 0
     current_record.num_of_tasks = total_num_of_email
     db.session.commit()
-    # Tried to add the process of building up htmls into threadpool, failed.
-    # with ThreadPoolExecutor(max_workers=10) as executor_building_html:
-    #     for group in group_col:
-    #         students_email = select_students_by_group(group, group_worksheet)
-    #         # grade_of_group = select_row_by_group_id(group)
-    #         # students_in_one_group = get_students_by_group(group_worksheet, students_worksheet)
-    #         # load download_page.html and store it to 'part' which will be attached to message in mail
-    #         file_name = "{}_{}_{}.html".format(project.project, evaluation_name, group)
-    #         path_to_html = "{}/{}".format(path_to_load_project, file_name)
-    #         if os.path.exists(path_to_html):
-    #             os.remove(path_to_html)
-    #         with open(path_to_html, 'w') as f:
-    #             executor_building_html.submit(f.write(download_page(project.project_id, evaluation_name, group, "normal", show_score)))
-    #
-    # with ThreadPoolExecutor(max_workers=10) as executor_sending:
-    #     for group in group_col:
-    #         students_email = select_students_by_group(group, group_worksheet)
-    #         file_name = "{}_{}_{}.html".format(project.project, evaluation_name, group)
-    #         path_to_html = "{}/{}".format(path_to_load_project, file_name)
-    #         if os.path.exists(path_to_html):
-    #             os.remove(path_to_html)
-    #         task_status = executor_sending.submit(send_emails_to_students, group, project, evaluation_name, from_email,
-    #                                               path_to_html, students_email, current_record)
-
-    with ThreadPoolExecutor(max_workers=10) as executor_sending:
-        for group in group_col:
+    
+
+    #Local Tesing
```

Comment:
replace this comment with a more detailed description about what "send emails" does when not on the main server